src/GitTabExpansion.ps1

```diff
@@ -188,12 +188,13 @@ function script:gitAliases($filter) {
                 $alias
             }
         }
-    } | Sort-Object
+    } | Get-Unique | Sort-Object
```

Comment:
It is probably a bit better to use `Sort-Object -Unique`.  And yes, that is available on v2 (just checked).